src/styles/2-components/_header.scss

```diff
@@ -0,0 +1,13 @@
+.header {
+  box-shadow: 0 2px 4px 0  rgba(0, 0, 0, 0.05);
+}
+
+.logo__img {
+  display: block;
+}
+
+.header-container {
+  @include flex-align(space-between);
+  box-sizing: border-box;
+  font-weight: 500;
+}
```

Comment:
If it is a separate block "logo", move to the file logo, if it's related to the header the class name should be .header__logo-img. The same for .header-container